packages/cli/test/functional/test_site_templates/test_default/expected/stylesheets/main.css

```diff
@@ -110,10 +110,6 @@ mark {
     #site-nav {
         display: none;
     }
-
-    footer {
-        margin-left: 60px;
-    }
 }
```

Comment:
Only the changes in this file need to be committed as modifying the css file, `packages/core/template/default/stylesheets/main.css`, will only affect this test file.

Reply:
Thanks for checking this for me! I've reverted and pushed changes to just that file.